x-pack/plugin/mapper-unsigned-long/src/test/java/org/elasticsearch/xpack/unsignedlong/UnsignedLongFieldMapperTests.java

```diff
@@ -365,6 +370,38 @@ protected IngestScriptSupport ingestScriptSupport() {
         throw new AssumptionViolatedException("not supported");
     }
 
+    @Override
+    protected List<NumberFieldMapperTests.OutOfRangeSpec> outOfRangeSpecs() {
+        return Collections.emptyList(); // unimplemented
+    }
+
+    @Override
+    public void testIgnoreMalformedWithObject() {} // unimplemented
+
+    @Override
+    public void testAllowMultipleValuesField() {} // unimplemented
+
+    @Override
+    public void testScriptableTypes() {} // unimplemented
+
+    @Override
+    protected Number missingValue() {
+        return 123L;
+    }
+
+    @Override
+    protected Number randomNumber() {
+        if (randomBoolean()) {
+            return randomLong();
+        }
+        if (randomBoolean()) {
+            return randomDouble();
+        }
+        assumeFalse("https://github.com/elastic/elasticsearch/issues/70585", true);
+        // TODO: unsigned long has a max value of (2^64)-1, but it can't be used in the randomDoubleBetween method because it's too large
+        return randomDoubleBetween(0L, Long.MAX_VALUE, true);
+    }
```

Comment:
Still need to resolve this one as well, was thinking of making a new method that supports larger numbers but maybe there's an easier way to support this for unsigned longs?